apps/web/src/stores/usedTrack.ts

```diff
@@ -58,12 +59,22 @@ export const useUsedTrackStore = defineStore('usedTrack', () => {
   })
 
   const completedQuestionnaireData = computed<(string | number | object | undefined)[]>(() => {
-    return completedUsedTracks.value
+    const data = completedUsedTracks.value
       .map((usedTrack: UsedTrack) => {
         return usedTrack.selected?.map((optionSelected) => toRaw(optionSelected.questionnaireData))
       })
       .filter((questionnaireDatum) => questionnaireDatum?.length)
       .flat(1)
+
+    if (CompanyDataStorage.hasSiret()) {
+      data.push(CompanyDataStorage.getSiret() as QuestionnaireData)
```

Comment:
Remarque /question : je trouve ça bizarre comme syntaxe. Tu ne peux pas juste écrire data[key]= ?

Reply:
ici getSiret() nous retourne un objet : ```{
  "siret": "84026120000017",
  "codeNAF": "49.41B",
  "codeNAF1": "H",
  "ville": "ARGENTEUIL",
  "codePostal": "95100",
  "region": "Île-de-France",
  "legalCategory": "5710",
  "denomination": "REZ",
  "secteur": "Transports routiers de fret de proximité",
  "creationDate": "2018-04-20"
}```
du coup pour push il faudrait que j'itère sur chacune des prop pour l'ajouter à data